src/main/kotlin/org/jitsi/videobridge/util/ArrayTracker.kt

```diff
@@ -35,9 +35,15 @@ open class ArrayTracker<T>(
      */
     private var head = -1
 
+    /**
+     * The numerically highest index value seen, or -1.
```

Comment:
Hm, I guess what I had in mind was something more about what an index represents.  If this is as detailed as it can get (because the class is generic) then I can see why you left them off :)